### ISSUE TITLE
Extract Annotation option disappear

### ISSUE THREAD

**Issue by pmtang** [jlegewie/zotfile#605]
zotero 6.0.4
After every restart of zotero, the "Extract Annotations" option will only pop out on the first time you do right klick on items. After that, this option will just disappear.

**Comment by blankroom** [comment]
![image](https://user-images.githubusercontent.com/12554452/164757578-48d66afb-41fe-4de1-87d2-73f288d3cf4d.png)
The option never pop out even if I restart or reinstall the plugin.

**Comment by Servinjesus1** [comment]
I am also having this issue. A workaround is to tell Zotfile to automatically extract annotations when getting PDFs from tablet, but that requires setting that up and modifying a PDF every time you want to extract (or re-extract) annotations.

**Comment by Servinjesus1** [comment]
It looks like it's a Zutilo issue. Disable that and reload to see the menu item again, or you can assign a shortcut in Zutilo.

**Comment by qqobb** [comment]
> It looks like it's a Zutilo issue.

No, this has been removed from the context menu in ZotFile 5.1, which also provides Zotero 6 compatibility. See here:

https://forums.zotero.org/discussion/94128/zotfile-no-option-to-extract-annotations/p1

https://www.zotero.org/support/kb/zotfile_extract_annotations

> **Where did the Extract Annotations button go?**
> 
> “Extract Annotations” was a feature of the third-party ZotFile plugin. In Zotero 6, that feature has been replaced by Zotero's own advanced PDF functionality.
